com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ExtenderUtil.java

```diff
@@ -472,7 +497,11 @@ public static boolean hasNativeExtensions(Project project) {
         for (String p : paths) {
             File f = new File(p);
             if (f.getName().equals(ExtenderClient.extensionFilename)) {
-                return true;
+                ArrayList<String> siblings = new ArrayList<>();
+                project.findResourceDirs(f.getParent(), siblings);
+                if (siblings.stream().anyMatch(x -> x.endsWith("src") || x.endsWith("commonsrc"))) {
+                    return true;
+                }
```

Comment:
We shouldn't "support" an ext.manifest at top level, that's what we have "game.appmanifest" for.
However, we don't want to get a java crash/callstack just because someone put such a file there.